Projects/TEAM-2/Tsihanchuk/Tasks/Task_202/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_202
+{
+    internal class Program
+    {   //2. Напишіть програму, яка переводить гривні в $, Є.
+        static void Main(string[] args)
+        {
+            float usd = 0;
+            Console.Write("USD: ");
```

Comment:
If you convert grivnas to dollars or euros - you have to write here Console.Write("UAH: ");